### ISSUE TITLE
Build Zodiac Square

### ISSUE THREAD

**Issue by feend78** [evennia/ainneve#89]
The existing build scripts contain central Riverport, east of the Kai River. To the west of the bridge where new players begin their quest, is _Zodiac Square_. 

This description is based on the world building document (Issue #58). The main road of Riverport runs through the square, and at its center stands a fountain. In the immediate square area are four shops:

* General Goods Shoppe
* AEther Shoppe
* Kaiwa Bazaar
* "Dancing Dragon" Inn

To the west of the square along the main road is the _Sosen Temple_ that could also be included.

The first three can likely be built using the `@buildshop` command once it is merged from PR #88. The _Dancing Dragon_ and _Sosen Temple_ will likely be more specialized for quests or narrative, and need to be adjusted later.

**Comment by maxsond** [comment]
Is there anything more to this than the build scripts for the rooms? I can't commit to doing this just yet because of holiday plans, but just checking if I have the right idea of the scope.

**Comment by feend78** [comment]
Yes, this is primarily an area building task, though it could certainly include some coding if your imagination takes off and you want to implement some custom object behavior.

**Comment by stefanludlow** [comment]
Feend, 
After discussing with Griatch, thought about compacting Riverport a bit. I've made the map and have the batchcommand file just about ready if you approve. I've kept as much as possible still in line with the original docs for quest purposes, with a few additions for some minor hooks: 

http://imgur.com/a/z6tag

**Comment by j-f1** [comment]
@stefanludlow Why don’t you put that into a PR so people can look at it?

<sub>Tip: if you mention `closes #89` in the PR, this issue will automatically be closed when the PR is merged.</sub>

**Comment by stefanludlow** [comment]
That would make too much sense! Will review what I have and either push a pr or paste here

**Comment by j-f1** [comment]
@stefanludlow Have you had a chance to look at this yet?

**Comment by stefanludlow** [comment]
I have not, and am unfortunantely internal atm without the pc that has
these on the actual db. That said, I have the text of them, which is here:

This is the building file for central.ev, it just needs the placeholder
descs replaced, which can be done further on.

https://pastebin.com/Yjgeh9jc

**Comment by j-f1** [comment]
I’m working on the file that @stefanludlow posted, and I should be posting a PR today.
